lib/monitoring/aws-kinesis/KinesisFirehoseMonitoring.ts

```diff
@@ -103,6 +116,51 @@ export class KinesisFirehoseMonitoring extends Monitoring {
       this.addAlarm(createdAlarm);
     }
 
+    for (const disambiguator in props.addIncomingBytesExceedThresholdAlarm) {
+      const alarmProps =
+        props.addIncomingBytesExceedThresholdAlarm[disambiguator];
+      const createdAlarm =
+        this.kinesisAlarmFactory.addFirehoseStreamExceedSafetyThresholdAlarm(
+          this.incomingBytesToLimitRate,
+          "IncomingBytes",
+          "BytesPerSecondLimit",
+          alarmProps,
+          disambiguator
+        );
+      this.recordCountAnnotations.push(createdAlarm.annotation);
```

Comment:
Do these new metrics make sense to have on the graph that shows a count of records? They seem to be more relevant for the other limits graph.

Reply:
thanks for the feedback, they are now added correctly to the limit widget